### PR TITLE
[gsoc] Conclusion Blog Post

### PR DESCRIPTION
**Pull request by lenary**
This is the blog post, as translated from the Google Doc.

### CODE REVIEW

File: site/content/blog/gsoc-2020-conclusion.md
Comment (vogelpi) [inline site/content/blog/gsoc-2020-conclusion.md:41]: @GregAC it would be great if you could have a quick look at this and make sure it accurately describes the current status of the codebase.

**Review by vogelpi — COMMENTED** [review]
Thanks @lenary for correcting and transferring this into a PR. This looks good to me, but the part on Flavien's project should also be cross-checked by @GregAC who has a better overview of what the final project status is.

File: site/content/blog/gsoc-2020-conclusion.md
Comment (GregAC) [inline site/content/blog/gsoc-2020-conclusion.md:41]:
```suggestion
All of Flavien's code, including documentation and a Verilator testbench can be found in [our
`gsoc-sim-mem` GitHub repository](https://github.com/lowRISC/gsoc-sim-mem).
```

FPGA stuff isn't quite working yet, there's some integration documentation but there's an outstanding issue so it still doesn't work. I think simplest to not mention the FPGA side of thing here.

File: site/content/blog/gsoc-2020-conclusion.md
Comment (GregAC) [inline site/content/blog/gsoc-2020-conclusion.md:34]:
```suggestion
by a separated Delay Calculator block that emulates a memory request scheduler and controller.
```

**Review by GregAC — CHANGES_REQUESTED** [review]

**Comment by lenary** [comment]
Ok, this needs review from @asb and then it can go forward. I'll let him merge when he's back in the office.

**Review by GregAC — APPROVED** [review]

**Comment by GregAC** [comment]
I did just force push a change to give it today's date rather than last week's date, but looked like I missed the merge. Probably not a big issue just felt it's a bit weird for posts to appear with dates in the past.

**Comment by asb** [comment]
I agree - I actually pushed a commit a bit before your message that fixes the publish date to today :)

**Pull request merged**
